components/dropDown/dropDown.tsx

```diff
@@ -34,31 +34,39 @@ export type DropDownItem = {
   isDelete?: boolean;
   status?: string;
 };
+export type HandleClickProps = {
```

Comment:
define this interface inside /utils/appTypes.tsx
and use everywhere

Reply:
Done